### DISCUSSION TITLE
What is the format of the RDS Data API schema parameter

### DISCUSSION THREAD

**Discussion by alextriaca** [aws/aws-cli#7249] [Q&A]
The batch-execute-statement function in the RDS Data API has a `schema` argument that can be passed in (see below) however it doesn't seem to have any affect on the output. Does anyone know what this field should contain? The docs are very vague and simply say "The name of the database schema". Any help is much appreciated!

```
  batch-execute-statement
--resource-arn <value>
--secret-arn <value>
--sql <value>
[--database <value>]
[--schema <value>]
[--parameter-sets <value>]
```

**Comment by tim-finnigan** [comment]
Hi @alextriaca thanks for reaching out. Interestingly in the ExecuteStatement [documentation](https://docs.aws.amazon.com/rdsdataservice/latest/APIReference/API_ExecuteStatement.html#API_ExecuteStatement_RequestBody) there's a note saying:
>Currently, the `schema` parameter isn't supported. 

I wonder if that note should be in the BatchExecuteStatement [documentation](https://docs.aws.amazon.com/rdsdataservice/latest/APIReference/API_BatchExecuteStatement.html) as well. I'll try reaching out to the RDS data team for more information.

**Reply by alextriaca** [reply]
Thanks @tim-finnigan, I wondered the same. The interesting difference I've noticed is that the ExecuteStatement function returns a `columnMetadata` field that describes what you're getting back. This isn't present on the BatchExecuteStatement so assumed thats where it may come into play. I have a sneaking suspicion you are right though and this is just a dead argument.
